stable/concourse/templates/web-deployment.yaml

```diff
@@ -30,15 +30,6 @@ spec:
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
           args:
             - "web"
-            {{- if eq (default "" .Values.credentialManager.vault.authBackend) "approle" }}
-            - "--vault-auth-param"
-            - "role_id=$(CONCOURSE_VAULT_APPROLE_ID)"
-            - "--vault-auth-param"
-            - "secret_id=$(CONCOURSE_VAULT_APPROLE_SECRET_ID)"
-            {{- end }}
-            {{- if .Values.credentialManager.kubernetes.enabled }}
-            - "--kubernetes-in-cluster"
-            {{- end }}
             {{- if .Values.credentialManager.ssm.enabled }}
             - "--aws-ssm-region"
             - "$(CONCOURSE_AWS_SSM_REGION)"
```

Comment:
Could we get rid of these cmd line args and use the env var `CONCOURSE_AWS_REGION` instead? From the output of `concourse web --help` it looks like that env var is used in both AWS SSM and Secretsmanager cases. Other than that we're looking good!

Reply:
According to the binary I have, the variable it wants to use is `AWS_REGION` (not `CONCOURSE_AWS_REGION`, which would be a little nicer), which seemed like a "heavy" env var to set if you want to configure either the `--aws-secretsmanager-region` or `--aws-ssm-region`. It would mean that configuring one implicitly (and perhaps surprisingly) configures the other one. This is "expected behavior" from Concourse's perspective, but it felt weird to support it, which is why I left it.

If my argument isn't convincing, just let me know and I'll change it :)

Reply:
@wendorf you're right it's `AWS_REGION`, and more importantly, it should be avoided due to this unfortunate issue: https://github.com/concourse/concourse/issues/2191. Lets stick with the cmd line args, but would it be possible to pass the value directly into the args rather than using an env var to propagate it?

Reply:
Sounds good. Just pushed the change.